Lib/argparse.py

```diff
@@ -1697,6 +1697,28 @@ def add_mutually_exclusive_group(self, *args, **kwargs):
         return super().add_mutually_exclusive_group(*args, **kwargs)
 
 
+def _prog_name(prog=None):
+    if prog is not None:
+        return prog
+    arg0 = _sys.argv[0]
+    try:
+        modspec = _sys.modules['__main__'].__spec__
+    except KeyError:
+        # possibly PYTHONSTARTUP or -X presite
+        # no good answer here
+        return _os.path.basename(arg0)
```

Comment:
```suggestion
        # no good answer here, so fall back to the default
        modspec = None
```

Cosmetic, but emphasises that a missing `__main__` module and a missing `__main__.__spec__` attribute have the same result.